specification/toolingservice/data-plane/1.0/toolingservice.json

```diff
@@ -0,0 +1,755 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Tooling Service Client",
+    "description": "Client that can be used to access Tooling services to run linter, static and api validations.",
+    "version": "1.0"
+  },
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "host": "<TBD>",
+  "basePath": "/api/",
+  "schemes": [
+    "https", "http"
+  ],
+  "paths": {
+    "/tools/lint": {
+      "post": {
+        "tags": [
+          "Linter"
+        ],
+        "operationId": "Tools_Linter",
+        "description": "Get Linter Result",
+        "parameters": [
+          {
+            "name":"repository",
+            "in": "body",
+            "type": "string",
+            "required": true,
+            "description": "Name of the repository"
+          },
+          {
+            "name":"branch",
+            "in": "body",
+            "type": "string",
+            "required": true,
+            "description": "Name of the specific branch in the repository."
+          },
+          {
+            "name":"configPath",
+            "in": "body",
+            "type": "string",
+            "required": true,
+            "description": "Configuration path of the file."
+          },
+          {
+            "name":"apiTag",
+            "in": "body",
+            "type": "string",
+            "required": true,
+            "description": "API Tag to be validated."
+          },
+          {
+            "name":"commitHash",
+            "in": "body",
+            "type": "string",
+            "description": "Commit hash to be validated."
+          },
+          {
+            "name":"mode",
+            "in": "query",
+            "type": "string",
+            "description": "Mode"
+          }          
+        ],
+        "responses": {
+          "400": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/ErrorObject",
+              "description": "Missing Parameters"
+            }
+          },
+          "201": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/LinterValidationResult",
+              "description": "Linter Result"
+            }
+          },
+          "200": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/LinterValidationResult",
+              "description": "Linter Result"
+            }
+          }
+        }
+      },
+      "x-ms-long-running-operation": true
+    },
+    "/tools/validation/semantic": {
+      "post": {
```

Comment:
@vladbarosan  Remember that while calculating validationId the files have to be sorted so the op remains idempotent (possibly)